### PR TITLE
Table element implementation example

### PR DESCRIPTION
**Pull request by zaliqarosli**
LORIS Medical History instrument

medicalhistory_page1
medicalhistory_page2
medicalhistory_page3

### CODE REVIEW

**Comment by zaliqarosli** [comment]
issue to discuss: How would we reuse fields while still ensuring unique names to store field's data in the database?

**Comment by akeshavan** [comment]
This is a good point! In Mindlogger, we are storing responses to an activity together (@shnizzedy can say more about it), so in your table example, I think we'd POST a response that would look like:

```javascript
{
   'uri/to/table/item': [
    {'uri/to/incident_item': response, 'uri/to/age/item': 20, 'uri/to/hospitalized/item': 1},
    {...}
  ],
  'uri/to/other/item': some_response
}
```

**Comment by zaliqarosli** [comment]
@ridz1208 please review

**Comment by zaliqarosli** [comment]
would love to also hear what you think about the "layout element" implementations i.e. table type @sanuann @akeshavan

**Comment by satra** [comment]
@zaliqarosli - the layout element attribute appears to very closely add ui components to the underlying schema. can we instead think about how this would be handled under the `ui` node at the activity level itself? just as we are doing for other things there.

the reason i want to pull the layout element out is that some other person may want to create an activity that has all the same items but with a different layout (for example a table for a set of common yes know questions (the phq9 could be rendered as a table).

if we can encode it under `ui` then individuals can easily vary the layout bits without changing or creating new sub activities.

i'm not against the possibility, but i would just like to see a good rationale for encoding layout into the schema sub nodes as opposed to controlling this at the activity node level.

**Comment by zaliqarosli** [comment]
@satra thanks for your comment, i will look into it! :)

**Comment by zaliqarosli** [comment]
@satra - please see me last three commits. this change ([link](https://github.com/ReproNim/schema-standardization/pull/140/commits/3a96e5ca51cf8cf56744bedeac57efa6ceaab98e#diff-58ce0e61997480bcfa3502ed735d0e52)) should work for both of us

**Comment by zaliqarosli** [comment]
@akeshavan i have rebased! no more conflicts

File: activities/MedicalHistory-LORIS/medicalhistory_sectionD.jsonld [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    "@type": "https://raw.githubusercontent.com/ReproNim/schema-standardization/master/schemas/Activity.jsonld",
+    "@id": "medicalhistory_sectionD",
+    "skos:prefLabel": "Section D of Medical History Instrument",
+    "skos:altLabel": "medicalhistory_sectionD",
+    "schema:description": "Section D of Medical History Instrument",
+    "schema:schemaVersion": "0.0.1",
+    "schema:version": "0.0.1",
+    "preamble": "D. Neurological",
+    "ui": {
+        "inputType": "section",
```
Comment (akeshavan) [inline activities/MedicalHistory-LORIS/medicalhistory_sectionD.jsonld]: inputType = "table" ?

File: activities/MedicalHistory-LORIS/medicalhistory_sectionD.jsonld [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+            "medicalhistory_21b_incident",
+            "medicalhistory_21b_hospitalized",
+            "medicalhistory_21b_age",
+            "medicalhistory_21c_incident",
+            "medicalhistory_21c_hospitalized",
+            "medicalhistory_21c_age",
+            "medicalhistory_21d"
+        ],
+        "shuffle": false,
+        "render": {
```
Comment (zaliqarosli) [inline activities/MedicalHistory-LORIS/medicalhistory_sectionD.jsonld]: to follow up from our conversation, i understand that we want activities with different renderings i.e. table vs not table to be separate activities. but then why is this way of doing it (with the render key) problematic? even though i prefer having the inputType: "table" implementation, i'm trying to understand why this way of implementing is not suitable @satra @akeshavan

Comment (akeshavan) [reply activities/MedicalHistory-LORIS/medicalhistory_sectionD.jsonld]: I think "render" is fine! If we know its a table, we'll (in theory) know to look at "render"

**Comment by zaliqarosli** [comment]
hi @satra @akeshavan i've changed the inputType=table activity back to what we discussed. please review the latest commit and comment. thanks.

**Pull request merged**